### PR TITLE
Add the tags variable to the ECR repository object

### PR DESCRIPTION
**Pull request by vitroth**
## What
This passes through the existing tags variable to the ECR resource
so that the repository is tagged with any user requested settings.

## Why
Add support for tagging of ECR repositories with user provided metadata

### CODE REVIEW

**Comment by sarkis** [comment]
@vitroth have you tested the case where empty map is passed in for `var.tags`? My concern here is if TF is handling that properly in latest 0.11.x.

**Comment by vitroth** [comment]
@sarkis Oh, good thinking...  I just tested it and passing the (default0 empty map works.

File: main.tf [hunk 1]
```diff
@@ -18,6 +18,7 @@ module "label" {
 resource "aws_ecr_repository" "default" {
   count = "${var.enabled == "true" ? 1 : 0}"
   name  = "${var.use_fullname == "true" ? module.label.id : module.label.name}"
+  tags  = "${var.tags}"
```
Comment (aknysh) [inline main.tf]:
```suggestion
  tags  = "${module.label.id.tags}"
```

The `label` module will merge the provided tags with some generated tags (e.g. Name, NameSpace, Stage).
This is our pattern of naming and tagging resources.

Comment (vitroth) [reply main.tf]: Updated, thanks.

**Comment by vitroth** [comment]
> Also, not related to this PR, but please rebuild README by executing these commands:

FYI, I just ran those steps and there were no changes to the README

**Pull request merged**